client/cl_main.lua

```diff
@@ -0,0 +1,322 @@
+local jobs = exports.qbx_core:GetJobs()
+local gangs = exports.qbx_core:GetGangs()
+local DynamicMenuItems = {}
+
+function CommaValue(amount)
+    local numChanged
+
+    repeat
+        amount, numChanged = string.gsub(amount, '^(-?%d+)(%d%d%d)', '%1,%2')
+    until numChanged == 0
+
+    return amount
+end
+
+---@return table
+function FindPlayers()
+	local playerCoords = GetEntityCoords(cache.ped)
+    local closePlayers = lib.getNearbyPlayers(playerCoords, 10, false)
+    for _, v in pairs(closePlayers) do
+        v.id = GetPlayerServerId(v.id)
+    end
+	return lib.callback.await('qbx_management:server:getplayers', false, closePlayers)
+end
+
+---@param type 'job'|'gang'
+RegisterNetEvent('qbx_management:client:SocietyMenu', function(type)
+    local amount = lib.callback.await('qbx_management:server:getAccount', false, QBX.PlayerData[type].name)
+    local SocietyMenu = {
+        {
+            title = 'Deposit',
+            icon = 'fa-solid fa-money-bill-transfer',
+            description = 'Deposit Money',
+            event = 'qbx_management:client:SocietyDeposit',
```

Comment:
Here and downwords there's a bunch of "type" that we need to have renamed to group as you have seen in the other messages.

Reply:
Done.